### ISSUE TITLE
Exact list of images that were used in training and development

### ISSUE THREAD

**Issue by dreichCSL** [pzzhang/VinVL#29]
Is there an exact list of image IDs that were used in training and tuning of hyperparameters (ie images that were part of any dev sets that were potentially used)?

While this might list might be useful in general, the reason for me specifically is that I want to verify whether or not the GQA features provided as download were produced by a model that was trained with images from the GQA balanced validation set (or GQA testdev set). 

Thanks!
